winlogbeat/sys/eventlogging/zsyscall_windows.go

```diff
@@ -96,6 +97,8 @@ func _ReadEventLog(eventLog Handle, readFlags EventLogReadFlag, recordOffset uin
 	r1, _, e1 := syscall.Syscall9(procReadEventLogW.Addr(), 7, uintptr(eventLog), uintptr(readFlags), uintptr(recordOffset), uintptr(unsafe.Pointer(buffer)), uintptr(numberOfBytesToRead), uintptr(unsafe.Pointer(bytesRead)), uintptr(unsafe.Pointer(minNumberOfBytesNeeded)), 0, 0)
 	if r1 == 0 {
 		if e1 != 0 {
+			// if an error is apeared, wait one second
+			time.Sleep(time.Second)
```

Comment:
I am not sure why this line is require as I did not see its use from event log plugin perspective but as this was a change done in previous elastic-beats version that is why I kept it.